tests/test-mover-included.c

```diff
@@ -1,8 +1,11 @@
+// clang-format off
 /*
 run:
   status: 24
   stdout: 14 10
 */
+//clang-format on
```

Comment:
Missing space after `//`.

Reply:
I am amazed at the eye for detail that you have. I didn't notice that and I went over all of the files just to check if they are ok.

Reply:
I can't help it ;)